docs/ref/forms/api.txt

```diff
@@ -700,6 +700,29 @@ example, in the ``ContactForm`` example, the fields are defined in the order
 ``subject``, ``message``, ``sender``, ``cc_myself``. To reorder the HTML
 output, just change the order in which those fields are listed in the class.
 
+There are three options to specify the order:
+
+.. versionadded:: 1.8
+
+.. attribute:: Form.field_order
+
+By default, ``field_order`` is not defined, which retains the order in which
+you define them in your class (default order).
+If ``field_order`` is a list of field names, the fields are ordered as specified by the list,
+and remaining fields are appended according to the default order.
+Unknown field names in the list are ignored.
```

Comment:
did you see a use case for silently ignoring fields? seems to me it would be better to fail loudly.